Small tweaks to highlight our ad services (#377)
readthedocs/ethicalads.io: content/pages/advertising-faq.md

Absolutely. You can contact us anytime, but an ad specialist will proactively reach out
a few weeks into your campaign with suggestions.
We also offer advice on crafting ads that resonate with our audience.
See our [advertising guide]({filename}/pages/learning-hub/advertiser-guide.md) for more details
and our [creative specs guide]({filename}/pages/learning-hub/ad-design-and-specs.md) has a number of tips and tricks about what works.

We can work with you to ensure your campaign is successful.
If you have creatives you're running on other networks, we can help you adapt them to our ad formats.

> If you need more help creating ads that convert, let us know.

Author: ```suggestion
If you need more help creating ads that convert, we can help with that.
```
